test/hotCases/chunks/accept-system-import/index.js

```diff
@@ -16,3 +16,22 @@ it("should import a changed chunk", (done) => {
 		}).catch(done);
 	}).catch(done);
 });
+
+it("should import a changed chunk using import.meta.hot.accept API", (done) => {
```

Comment:
You can't put multiple tests in a hotCase. Create a new folder.

each `NEXT` goes to the next hmr update and there is no way back for the next test.